authors/apps/authentication/tests/test_base.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework.test import APIClient,APITestCase
+from django.urls import reverse
+from ...authentication.models import User
+
+
+class BaseTest(APITestCase):
+    """
+    sets up a base class upon which all our test depends on
+    """
+    def setUp(self):
+        self.clien= APIClient()
```

Comment:
@anyatibrian in you BaseTest class under the method `def setUp` I notice you creating an instance of the `APIClient()` as `self.clien=APIClient()` and there is a typo it should be `self.client=APIClient()`, as you continue using `self.client` when mocking your tests.

Reply:
its done, i just realised it few minutes ago